pkg/controller/machine/controller.go

```diff
@@ -128,14 +143,38 @@ func (c *MachineControllerImpl) Reconcile(machine *clusterv1.Machine) error {
 	}
 	if exist {
 		glog.Infof("Reconciling machine object %v triggers idempotent update.", name)
-		return c.update(m)
+		err = c.update(m)
+		if err != nil {
+			requeueErr, ok := err.(*RequeueAfterError)
```

Comment:
nit: combine with the next line:

`if requeueErr, ok := err.(*RequeueAfterError); ok { ... }`